### ISSUE TITLE
India: Investigate new municipal, state, and district admin data from DataMeet

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#636]
A new set of administrative municipality, state, and district data for India is available from DataMeet. [This article](http://thejeshgn.com/2017/02/03/community-created-free-open-maps-of-india/) outlines the types of data now available from the group.

The data is available under a [CC-BY 2.5 IN](https://creativecommons.org/licenses/by/2.5/in/). 

[Link](http://projects.datameet.org/Municipal_Spatial_Data/) to municipality data.
[Link](https://github.com/datameet/maps/tree/master/Districts) to state and district data.
[Link](http://projects.datameet.org/Municipal_Spatial_Data/) to the DataMeet GitHub page.

TO DO:

- [x] Investigate new `admin1` (States) data, update WOF records
  - WOF data was updated with new `admin1` data, see: be538aaf
- [x] Investigate new `admin2` (Districts) data, update WOF records
  - WOF data was updated with new `admin2` data, see [the Spelunker](https://whosonfirst.mapzen.com/spelunker/id/85632469/descendants/?exclude=nullisland&placetype=county#8/10.312/80.555)
- [ ] Investigate new "municipality" data, update WOF records
  - This data represents different placetypes - will update in a separate PR accordingly
- [ ] Investigate state assembly constituencies in India, create new WOF records
  - This data will likely go into a new repo, similar to the `whosonfirst-data-constituency-us` data repo
- [ ] Investigate other constituency level data in India
  - TBD

**Comment by tallytalwar** [comment]
👍 

Let me know if you need any help here.  Thanks

**Comment by tallytalwar** [comment]
Was able to convert some of shapefiles to geojson:

GeoJSON for all of state assembly constituencies in India (this is huge): https://gist.githubusercontent.com/tallytalwar/6390d2f5e005255867808b0e04a1add0/raw/8eeef95f751783112ad0b5fa212b66b35caf7bd4/India_AC.geojson

I extracted out one of the state (Punjab's) state assembly constituencies: https://gist.githubusercontent.com/tallytalwar/6390d2f5e005255867808b0e04a1add0/raw/8eeef95f751783112ad0b5fa212b66b35caf7bd4/Punjab_AC.geojson

I wrote a very crude python script to extract individual state's data from the huge india one. Let me know if you want to use it. :D

**Comment by stepps00** [comment]
Awesome, thanks! I'll start tackling this next week and let you know if I have any questions.

**Comment by tallytalwar** [comment]
Sure, I am also in middle of merging some of the election results from previous year in the geojson I created for the state of punjab.

Does wof care about having election results (old and new) in its dataset? we can also scrape election commision of india website directly (https://github.com/HindustanTimesLabs/eci-shape-scrape)

**Comment by stepps00** [comment]
It wouldn't be a priority to add those properties to WOF records, but it wouldn't hurt to add them if we have the data. We would need to make sure that the properties are indexed correctly, though, and we'd need to update our [properties repo](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties) and [sources repo](https://github.com/whosonfirst/whosonfirst-sources/tree/master/sources) with relevant pointers.

I'll ping you when I dive into the new India data.

**Comment by stepps00** [comment]
@tallytalwar - Spent a bit of time today with the DataMeet datasets, I've updated the initial issue comment. A few notes:

- I've updated WOF records in India at the `admin1` (region) and `admin2` (county) level with DataMeet data. The county data is accessible from the Spelunker and the region data is waiting to be imported. 
- The "municipality" data is scattered throughout India and represents different placetypes in Who's On First. At first glance, it looks like most of these features are not in Who's On First yet, so getting these features in is the next step. 
- We can import the India constituency data, but this will probably live in a repository similar to the [whosonfirst-data-constituency-us](https://github.com/whosonfirst-data/whosonfirst-data-constituency-us) repo.

To summarize, work at the region and county level has been done. Next steps are to import the various "municipality" records and having a think about how to track the constituency data. I'll update this issue as the work continues.

**Comment by tallytalwar** [comment]
Thanks @stepps00.

Will give this a look soon.

**Comment by stepps00** [comment]
I should note that the county-level data for India is sourced from `meso`, by way of `DataMeet`. 

See: [WOF record for Akola](https://whosonfirst.mapzen.com/spelunker/id/890502567/#9/20.7629/77.1547)

**Comment by stepps00** [comment]
Going to close this issue out, as we've harvested all of the openly licensed data from DATAMEET available. There is another non-DATAMEET dataset available with locality and neighbourhood place records, we can track importing that through a separate issue.
